src/cnn.py

```diff
@@ -68,26 +68,6 @@ def add_upsampling_block(input_layer, block_index, downsampling_conv2_layers):
     return BatchNormalization()(conv2)
 
 
-def get_masked_categorical_crossentropy(cdl_indices_to_mask):
-    def masked_categorical_crossentropy(y_true, y_pred):
-
-        # Used for pixel classifications: mask pixels whose class is in cdl_indices_to_mask
-        # Note: shapes are (batch, image_shape, image_shape, n_classes)
-        # TODO Speed this up, test it
-
-        mask = K.ones_like(y_true[:, :, :, 0])
-
-        for cdl_index in cdl_indices_to_mask:
-            cdl_index_indicators = K.cast(
-                K.equal(y_true[:, :, :, cdl_index], 1), "float32"
-            )
-            mask -= cdl_index_indicators
-
-        return K.categorical_crossentropy(y_true, y_pred) * mask
```

Comment:
I got errors here when running fit_model.py on this branch

Reply:
I replaced this with temporal sample weights (based on class):  see https://github.com/keras-team/keras/issues/3653#issuecomment-557844450